tests/conformance/configuration/configuration.go

```diff
@@ -142,13 +147,21 @@ func ConformanceTests(t *testing.T, props map[string]string, store configuration
 	processedC3 := make(chan *configuration.UpdateEvent, keyCount*4)
 
 	t.Run("init", func(t *testing.T) {
-		err := store.Init(context.Background(), configuration.Metadata{
+		// Initializing config updater. It has to be initialized before the store to create the table
+		err := updater.Init(props)
+		assert.NoError(t, err)
+
+		// Creating trigger for postgres config updater
+		if component == postgres {
+			err = updater.(*postgres_updater.ConfigUpdater).CreateTrigger(pgNotifyChannel)
+			assert.NoError(t, err)
+		}
+
+		// Initializing store
+		err = store.Init(context.Background(), configuration.Metadata{
 			Base: metadata.Base{Properties: props},
 		})
-		assert.Nil(t, err)
-		// Initializing config updater
-		err = updater.Init(props)
-		assert.Nil(t, err)
+		assert.NoError(t, err)
 	})
 
```

Comment:
Here you may also want to add a line that makes the conf test exit if the init failed. If the component couldn't be initialized, there's no point in trying to run the tests.

```go
if t.Failed() {
    t.Fatal("initialization failed")
}
```